### PR TITLE
Update LimitChunkCountPlugin docs

### PR DESCRIPTION
**Pull request by erykpiast**
Remove info about `minChunkSize` argument and refer to `MinChunkSizePlugin` plugin.

### CODE REVIEW

File: src/content/plugins/limit-chunk-count-plugin.md [hunk 1]
```diff
@@ -43,23 +44,7 @@ module.exports = {
...
-module.exports = {
-  // ...
-  plugins: [
-    new webpack.optimize.LimitChunkCountPlugin({
-      minChunkSize: 1000
-    })
-  ]
-};
-```
+To keep chunk size above the specified limit by merging chunks that are smaller than the limit, use [MinChunkSizePlugin)[/plugins/min-chunk-size-plugin]
```
Comment (EugeneHlushko) [inline src/content/plugins/limit-chunk-count-plugin.md]: Why is this needed here if its not done via this plugin anymore?

Comment (erykpiast) [reply src/content/plugins/limit-chunk-count-plugin.md]: Aren’t plugins a valid way of configuring Webpack anymore?

Comment (EugeneHlushko) [reply src/content/plugins/limit-chunk-count-plugin.md]: i mean if there is a plugin that does exactly this, move this info to its page?

Comment (erykpiast) [reply src/content/plugins/limit-chunk-count-plugin.md]: There is already such info on the page I linked to. I've added the link to make migration from old API to new one a bit easier. It's more like "hey, this feature was moved somewhere else, if you search for it, look at here".

Comment (EugeneHlushko) [reply src/content/plugins/limit-chunk-count-plugin.md]: Okay, that is still what i think is not really obvious from the text. The text says: "To do X, use plugin X", when you are saying it is intended to be "Hey, feature X has moved to plugin X". Or am i reading this wrong?

Comment (erykpiast) [reply src/content/plugins/limit-chunk-count-plugin.md]: You're reading it right. I'll try to make the message clearer :)

File: src/content/plugins/limit-chunk-count-plugin.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ module.exports = {
 
 ### `minChunkSize`
 
-To keep chunk size above the specified limit by merging chunks that are smaller than the limit, use [MinChunkSizePlugin)[/plugins/min-chunk-size-plugin]
+Keeping chunk size above the specified limit is no longer a feature of this plugin. Use [MinChunkSizePlugin)[/plugins/min-chunk-size-plugin] instead.
```
Comment (EugeneHlushko) [inline src/content/plugins/limit-chunk-count-plugin.md]: Worth mentioning a version since when that happened as a reference or is that sufficient enough? CC @montogeek

Comment (montogeek) [reply src/content/plugins/limit-chunk-count-plugin.md]: It is enough

**Comment by EugeneHlushko** [comment]
Thanks!
